src/nn/wasm/PreparedModel.js

```diff
@@ -44,6 +44,10 @@ export default class PreparedModel {
         this._nn_ops.set_gemm_context_threads_num(1);
     }
 
+    if (model._operands[modelInputs[0]].type === OperandCode.TENSOR_QUANT8_ASYMM) {
```

Comment:
What's the reason that only for `TENSOR_QUANT8_ASYMM`? What's the threads number for other types?

Reply:
Sorry this is a mistake caused by my carelessness. I refered to the previous code, but I didn't realize that cpu_backend_context is not only used in QUANT layer. Could you review the commit I made just now? Thanks